HW_27.10/HW_27.10/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "Path1": "books.json"
```

Comment:
why Path1?
looks silly, call it dataFilePath for example, so every new person who looks at your code will know